tests/requirements.txt

```diff
@@ -4,4 +4,5 @@
 cryptography<3.4;python_version<"3.7"
 pyproj~=2.2
 dash[dev,testing]>=1.3.1
-Pillow
\ No newline at end of file
+Pillow
+black
```

Comment:
Should specify an exact version of `black` to minimize the tests breaking on their own.